tests/testthat/test-utils-github.R

```diff
@@ -11,7 +11,7 @@ test_that("parse_github_remotes() works, on named list or named character", {
     bitbucket2 = "git@bitbucket.org:OWNER/REPO.git"
   )
   parsed <- parse_github_remotes(urls)
-  expect_equal(parsed$name, names(urls))
+  expect_named(urls, parsed$name)
```

Comment:
This one seems to change the intent of the expectation. As in, I just created `urls` by hand with specific names. So I'm testing something about `parsed`, not about `urls`. This probably still succeeds / fails appropriately, but it somehow feels off.

Reply:
Thanks for the feedback. I agree it's less readable, this is one of the defaults of `lintr` that I will maybe deviate from.